package.json

```diff
@@ -48,13 +48,13 @@
     "fuzzy": "^0.1.3",
     "git-remote-origin-url": "^3.1.0",
     "inquirer": "^7.3.3",
-    "inquirer-autocomplete-prompt": "^1.0.2",
-    "netlify-cli": "^2.59.0",
+    "inquirer-autocomplete-prompt": "^1.2.0",
+    "netlify-cli": "^2.65.0",
```

Comment:
Same here

Reply:
Which one `netlify-cli` latest is `2.65.0`

Reply:
Oh ok, this is good then :+1: